rs/memory_tracker/src/lib.rs

```diff
@@ -306,8 +306,14 @@ impl SigsegvMemoryTracker {
     pub fn expand(&self, delta: usize) {
         let old_size = self.area().size.get();
         self.area().size.set(old_size + delta);
-        self.accessed_bitmap.borrow_mut().grow(delta);
-        self.dirty_bitmap.borrow_mut().grow(delta);
+        let num_pages = delta / PAGE_SIZE;
```

Comment:
Can you please clarify this? I'm not sure I understand: before we were doing `.grow(delta)`, now we are doing `.grow(delta / PAGE_SIZE)`; are they equivalent?